documentation/JavaFiddle_Requirements.txt

```diff
@@ -58,5 +58,8 @@ Reviewboard functionality
 - add/edit/read comments
 - approve/reject
 
+[Tsanda Artyom] Ideas:
+- add possibility to import project from an existing one on the local machine
+- add wiget with javadocs where you can search class documentation you want 
```

Comment:
As I imagine it myself, it will be just like a man pages u can use if u forgot some class description(E.g. math). It has been already implemented in netbeans, eclipse and idea.